src/test/scala/webservice/CromIamApiServiceSpec.scala

```diff
@@ -1,19 +1,28 @@
 package webservice
 
 import akka.event.NoLogging
+import akka.http.scaladsl.model.{HttpRequest, HttpResponse}
 import akka.http.scaladsl.model.StatusCodes._
 import akka.http.scaladsl.testkit.ScalatestRouteTest
 import org.scalatest._
 
+import scala.concurrent.Future
 
-class CromIamApiServiceSpec extends FlatSpec with Matchers with ScalatestRouteTest with CromIamApiService {
-  override def testConfigSource = "akka.loglevel = WARNING"
-  def config = testConfig
+class CromIamApiServiceSpec extends FlatSpec with Matchers with CromIamApiService with ScalatestRouteTest {
+  override def testConfigSource = "akka.loglevel = DEBUG"
   val logger = NoLogging
 
-  "Stats endpoint" should "respond with internal server error" in {
+  override def cromwellInterface: String = ???
+  override def cromwellPort: Int = ???
```

Comment:
`???` is really meant as a WIP placeholder, these should probably be changed to throw `NotImplementedError`s